pyproject.toml

```diff
@@ -7,7 +7,7 @@ allow-direct-references = true
 
 [project]
 name = "multiversx-sdk-network-providers"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
We should bump the minor version, `0.8.0` (breaking change & we are still in `v0.*`).